### PR TITLE
CIV-11516 Content change for LiP Small Claims DQs

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-11516

### Change description ###

CUI R1 - Content change for LiP Small Claims DQs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/features/directionsQuestionnaire/witnesses/otherWitnesses/other-witnesses.njk [hunk 1]
```diff
@@ -23,6 +25,9 @@
       classes: "govuk-button append-row"
   }) }}
 {% endset %}
+{% set notificationHtml %}
```
Comment (portizdev) [inline src/main/views/features/directionsQuestionnaire/witnesses/otherWitnesses/other-witnesses.njk:28]: I think we use the same DQs for the claimant, does this change apply to both parties?

Comment (LeonardoPalmeiro) [reply src/main/views/features/directionsQuestionnaire/witnesses/otherWitnesses/other-witnesses.njk:28]: in the src/main/views/features/directionsQuestionnaire/support-required-list.njk you're name html of notification with another string
image

**Review by LeonardoPalmeiro — CHANGES_REQUESTED** [review]
please make sure you're using same formatting of the rest of the team

File: src/main/views/features/directionsQuestionnaire/witnesses/otherWitnesses/other-witnesses.njk (hunk 1 above)
Comment (jeswanth-hmcts) [reply src/main/views/features/directionsQuestionnaire/witnesses/otherWitnesses/other-witnesses.njk:28]: Added condition for the defendant only and changed the name to notificationHtml  to keep consistency

**Comment by jeswanth-hmcts** [comment]
> please make sure you're using same formatting of the rest of the team

fixed formatting issues

**Review by neeta-hmcts — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

**Review by LeonardoPalmeiro — DISMISSED** [review]

**Review by portizdev — DISMISSED** [review]

**Review by portizdev — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Pull request merged**